android/build.gradle

```diff
@@ -34,6 +44,18 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlinx-serialization'
 
+// Optionally, set any parameters to send to the plugin.
+try{
+    if(qtAndroidDir){
+        ext.gleanYamlFiles = [
+                "$rootDir/../../../glean/metrics.yaml",
+                "$rootDir/../../../glean/pings.yaml"
```

Comment:
All metrics currently have 

```
 send_in_pings:
      - main
```

This means they will not be added to either the default `events` ping or `metrics` ping. You can change that by adding `- default` to the list of pings there. I am not sure this will address the problem of sending things in the background though, see my main review comment :)